binding/abstract.go

```diff
@@ -127,6 +127,11 @@ func (*AbstractDUT) DialSSH(context.Context, SSHAuth) (SSHClient, error) {
 	return nil, errors.New("DialSSH unimplemented")
 }
 
+// DialGRPC returns an unimplemented error.
+func (*AbstractDUT) DialGRPC(context.Context, int, ...grpc.DialOption) (*grpc.ClientConn, error) {
```

Comment:
Where is the `AbstractDUT` binding used such that this implementation is needed? (given that this is not being added to the `DUT` interface, it seems like it shouldn't be required?)

Reply:
Ah, I see, you are extending the `DUT` interface -- sorry, I missed this.

Is this required? Do we expect every DUT to implement `DialGRPC` by default, or should we specifically handle this through having a "dialable DUT" that is defined elsewhere? (I think this is a discussion to have with the ONDATRA team -- I briefly had a conversation with @marcushines about it -- and I believe he had suggested not extending the base `binding`).